decentralizediso20022/src/main/resources/application.properties

```diff
@@ -1,7 +1,26 @@
 server.servlet.context-path=/api
 
-message.empty=The entity was empty
-message.bad-header=The entity had an invalid from or to header.
-message.bad-pacs=The entity was not a valid pacs message.
-message.500=Could not process the message.
-message.200=The message was processed.
\ No newline at end of file
+message.empty=The message was empty.
+message.bad-pacs=The message was not a valid pacs message.
+
+message.bad-from-header=The message's From tag "%s" did not match the certificate's common name "%s".
+message.bad-to-header=The message's To tag "%s" did not match the certificate's common name "%s".
+message.bad-recipient=Failed to retrieve the recipient from the message's To tag "%s".

```

Comment:
This file has weird line endings, could this be fixed before merging?

Reply:
Has been fixed